src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/MethodologyVersionRepository.cs

```diff
@@ -87,6 +86,7 @@ await _contentDbContext.Entry(methodology)
         {
             return await _contentDbContext
                 .MethodologyVersions
+                .Include(mv => mv.Methodology) // for MethodologyVersion.Slug
```

Comment:
I don't think this is necessary only because of the comment below, stating:

```
// EF cannot translate mv.Slug into a Queryable, so we have to do this...
```